### PR TITLE
[xml] Update german.xml

### PR DESCRIPTION
**Pull request by schnurlos**
Modified backspace description.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/german.xml [hunk 1]
```diff
@@ -1115,7 +1115,7 @@ Translation note:
 					<Item id="6310" name="Einrückung verwendet:"/>
 					<Item id="6311" name="Tab Zeichen"/>
 					<Item id="6510" name="Standard verwenden"/>
-					<Item id="6512" name="Rückschritttaste entfernt Einrückungen, anstatt einzelne Leerzeichen zu entfernen"/>
+					<Item id="6512" name="Die Rückschrittaste U+232B entfernt Einrückungen statt einzelne Leerzeichen"/>
```
Comment (MarkusBodensee) [inline PowerEditor/installer/nativeLang/german.xml:1118]: I would just call it Rücktaste which seems the most common for me, U+232B is imo not needed, so:

"Rücktaste entfernt Einrückungen anstatt einzelner Leerzeichen"

Comment (donho) [reply PowerEditor/installer/nativeLang/german.xml:1118]: Are there more German speakers who can provide the opinion regarding this change?

Comment (MarkusBodensee) [reply PowerEditor/installer/nativeLang/german.xml:1118]: LEO.org validates it, too. I think Rücktaste is the correct and most common word for the backspace key.

https://dict.leo.org/englisch-deutsch/backspace

grafik

**Comment by schnurlos** [comment]
Now waiting for https://github.com/notepad-plus-plus/notepad-plus-plus/pull/15351 getting committed.

File: PowerEditor/installer/nativeLang/german.xml (hunk 1 above)
Comment (CennoxX) [reply PowerEditor/installer/nativeLang/german.xml]: As a German I say Backspace-Taste (which is the name in the according Wikipedia article too). With Rücktaste I would know what is meant. Rückschritttaste is rather uncommon.

**Pull request closed without merging**